CHANGELOG.md

```diff
@@ -6,6 +6,11 @@
 
 - Added support for specifying clustering keys when saving dataframes using `DataFrame.save_as_table`.
 - Accept `Iterable` objects input for `schema` when creating dataframes using `Session.create_dataframe`.
+- Added support for creating a Snowpark session from a configuration file or environment variables.
```

Comment:
I think we need more documentation for this feature, currently I can only find it in connector's changelog. So I created https://snowflakecomputing.atlassian.net/browse/SNOW-910845, and we should do it before next snowpark release.